app/src/features/projects/components/ProjectFundingItemForm.test.tsx

```diff
@@ -41,11 +41,6 @@ const investment_action_category: IInvestmentActionCategoryOption[] = [
 ];
 
 describe('ProjectFundingItemForm', () => {
-  beforeAll(() => {
-    // ignore un-fixable warning about initial agency_id select value being 0 when it expects it to be > 1
-    jest.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
nice!

how did you get it working without setting that default 0 value?
is it because of the `('' as unknown) as number`?

Reply:
Yeah exactly.  Though its not a perfect solution in all cases.